### PR TITLE
Add GettingStart.md Chinese version

### PR DESCRIPTION
**Pull request by chenyang23333**
Thank you for your contribution to the MindYOLO repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindyolo/blob/master/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: GETTING_STARTED_CN.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  ```
+
+* 评估模型的精度：
+
+  ```
+  python test.py --config ./configs/yolov7/yolov7.yaml --weight=/path_to_ckpt/WEIGHT.ckpt
+  ```
+*注意：默认设备为Ascend，您可以指定'device_target'为Ascend/GPU/CPU。*
+* 有关更多选项，请参阅 `train/test.py -h`.
+
```
Comment (zhanghuiyao) [inline GETTING_STARTED_CN.md:66]: 单卡训练超参调整那个也加上

**Review by CaitinZhao — DISMISSED** [review]

File: GETTING_STARTED_CN.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      labels/
+        {train,val}2017/
+            00000001.txt
+            ...
+            # label files that are mentioned in the corresponding train/val2017.txt
+  ```
+  </details>
+
+* 在多卡NPU/GPU上进行分布式模型训练，以8卡为例:
+  ```
```
Comment (CaitinZhao) [inline GETTING_STARTED_CN.md:49]: 加空行

File: GETTING_STARTED_CN.md (hunk 1 above)
Comment (chenyang23333) [reply GETTING_STARTED_CN.md]: 已加上

File: GETTING_STARTED_CN.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+本文简要介绍MindYOLO中内置的命令行工具的使用方法。
+
+### 使用预训练模型进行推理
+
+1. 从[model zoo](MODEL_ZOO.md)中选择一个模型及其配置文件，例如， `./configs/yolov7/yolov7.yaml`.
+2. 从[model zoo](MODEL_ZOO.md)中下载相应的预训练模型权重文件。
+3. 使用内置配置进行推理，请运行以下命令：
+
+```
```
[the remark this answers is not in the document]
Comment (chenyang23333) [reply GETTING_STARTED_CN.md]: 已修改

File: GETTING_STARTED_CN.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+* 要在CPU上运行，请将device_target的值修改为CPU.
+* 结果将保存在`./detect_results`目录下
+
+### 使用命令行进行训练和评估
+
+* 按照YOLO格式准备您的数据集。如果使用COCO数据集（YOLO格式）进行训练，请从[yolov5](https://github.com/ultralytics/yolov5)或darknet准备数据集.
+  
+  <details onclose>
+
+  ```
```
[the remark this answers is not in the document]
Comment (chenyang23333) [reply GETTING_STARTED_CN.md]: 已修改

**Review by CaitinZhao — APPROVED** [review]

**Pull request merged**